server/models/Event.js

```diff
@@ -9,29 +9,27 @@ const EventSchema = new mongoose.Schema({
     type: String,
     require: true,
   },
-  startDate: {
+  initialDate: {
     type: Date,
     require: true,
   },
-  endDate: {
+  finalDate: {
     type: Date,
     require: true,
   },
-  startTime: {
-    type: String,
-    require: true,
-  },
-  recurringRate: {
-    type: String,
+  recurring: {
+    type: Boolean,
     require: true,
   },
-  recurringDates: {
-    type: Array,
-  },
-  endTime: {
-    type: String,
-    required: true,
-  },
+  recurringDates: [
```

Comment:
I agree with putting title, description, start, end in the recurring dates array. I still think title and description should also be stored outside the recurring dates array.